### ISSUE TITLE
Show more Integration Information

### ISSUE THREAD

**Issue by lpgagnon** [KSP-RO/ProgramsIssueTracker#11]
VAB's Integration Info window could use a few additions

- Recovery time (and cost?) - in the SPH initially, maybe only later in VAB
- Integration Cost: this got removed because it's now done via engineer upkeep cost, but that doesn't eliminate the existence of _a_ cost. The build time is already derived from the number of engineers; cost can be derived the same way (time/365 x engineercount x 1000)
- And with that we can get Total Cost (parts+integ+rollout/recovery) 
- For SPH: the engineer cap for the current vessel
- Show the active LC, make it a button to cycle through LCs

**Comment by NathanKell** [comment]
I don't think it makes sense to show costs here, I mean I guess we could show your budget delta over the time the thing is building? But certainly I wouldn't want to show the engineer salary for just that time in that LC since you'd be paying that cost regardless.

Recovery doesn't have a cost AFAIK.

Definitely should show cap for SPH, and I need to make the KCT button do hide/show like in other scenes (and agree a quick button would be nice).

**Comment by lpgagnon** [comment]
"you'd be paying that cost regardless": sure, unless the information helps me decide that no, this thing isn't worth building and it's time to move resources out of the LC

As I said, it's basically trivial to calculate this, so without the label, I'll just go on calculating it. But it's a step backwards as far as getting a good grasp of a vessel's "true cost". (from "have to manually add up parts, integ and rollout cost" to "have to manually calculate integ cost, then add them up")
